plugin/src/main/scala/higherkindness/mu/rpc/srcgen/SrcGenPlugin.scala

```diff
@@ -129,9 +129,10 @@ object SrcGenPlugin extends AutoPlugin {
     muSrcGenIdlType := IdlType.Unknown,
     muSrcGenIdlExtension := {
       muSrcGenIdlType.value match {
-        case IdlType.Avro  => "avdl"
-        case IdlType.Proto => "proto"
-        case _             => "unknown"
+        case IdlType.Avro    => "avdl"
+        case IdlType.Proto   => "proto"
+        case IdlType.OpenAPI => "open api"
```

Comment:
I'd avoid doing this as part of the PR. I don't think Open API is fully supported

Reply:
ah, okay!  Good to know I thought it was an oversight.